src/main/scala/com/iheart/workpipeline/akka/patterns/WorkPipeline.scala

```diff
@@ -131,35 +138,36 @@ object PushingWorkPipeline {
   val defaultBackPressureSettings = BackPressureSettings(
     maxBufferSize = 60000,
     thresholdForExpectedWaitTime = 1.minute,
-    maxHistoryLength = 10.seconds)
-
-
+    maxHistoryLength = 10.seconds
+  )
 
   case class Settings(workPipelineSettings: WorkPipeline.Settings, backPressureSettings: BackPressureSettings)
 
   val defaultSettings: Settings = Settings(WorkPipeline.defaultWorkPipelineSettings, defaultBackPressureSettings)
 
-
 }
 
-case class PullingWorkPipeline( name: String,
-                                iterator: Iterator[_],
-                                pipelineSettings: WorkPipeline.Settings,
-                                backendProps: Props,
-                                metricsCollector: MetricsCollector = NoOpMetricsCollector,
-                                resultChecker: ResultChecker) extends WorkPipeline {
+case class PullingWorkPipeline(
+  name: String,
+    iterator: Iterator[_],
+    pipelineSettings: WorkPipeline.Settings,
+    backendProps: Props,
+    metricsCollector: MetricsCollector = NoOpMetricsCollector,
+    resultChecker: ResultChecker
```

Comment:
this is some weird indentation -- I guess in this case there was a space after the `(` and scalariform handled it weirdly